[2023-07-13 17:03:14,706] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.cargar_datos manual__2023-07-13T16:58:41.910820+00:00 [queued]>
[2023-07-13 17:03:14,833] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.cargar_datos manual__2023-07-13T16:58:41.910820+00:00 [queued]>
[2023-07-13 17:03:14,842] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 17:03:14,847] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-13 17:03:14,851] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 17:03:15,126] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cargar_datos> on 2023-07-13 16:58:41.910820+00:00
[2023-07-13 17:03:15,298] {standard_task_runner.py:52} INFO - Started process 932 to run task
[2023-07-13 17:03:15,448] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'cargar_datos', 'manual__2023-07-13T16:58:41.910820+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmpeyq_jjpr', '--error-file', '/tmp/tmpg2o0uf18']
[2023-07-13 17:03:15,461] {standard_task_runner.py:80} INFO - Job 35: Subtask cargar_datos
[2023-07-13 17:03:16,507] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.cargar_datos manual__2023-07-13T16:58:41.910820+00:00 [running]> on host bd4a51b00e09
[2023-07-13 17:03:17,625] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=cargar_datos
AIRFLOW_CTX_EXECUTION_DATE=2023-07-13T16:58:41.910820+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-13T16:58:41.910820+00:00
[2023-07-13 17:03:17,631] {logging_mixin.py:115} INFO - creando conexion
[2023-07-13 17:03:25,010] {script_etl.py:20} INFO - DB_USER probando usuario 2: bapintor_coderhouse
[2023-07-13 17:03:25,017] {logging_mixin.py:115} INFO - usuario probando 2 bapintor_coderhouse
[2023-07-13 17:17:39,128] {python.py:173} INFO - Done. Returned value was: None
[2023-07-13 17:17:39,886] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=cargar_datos, execution_date=20230713T165841, start_date=20230713T170314, end_date=20230713T171739
[2023-07-13 17:17:40,436] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-13 17:17:40,691] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
